CONTRIBUTING.md

```diff
@@ -3,25 +3,35 @@
 We rely on pull requests to keep this project maintained. By participating in this project, you
 agree to abide by the VMware [code of conduct](CODE-OF-CONDUCT.md).
 
+## Setup 
+
 Fork, then clone the repo:
 
     git clone git@github.com:carbonblack/carbon-black-cloud-sdk-python.git
 
-Set up your machine:
+It is recommended to use [virtualenv](https://virtualenv.pypa.io/en/latest/) to set up the project. Once you have that
+you can use it to generate a virtual environment, activate it and install the package.
 
-    python3 -m virtualenv ./venv
-    source venv/bin/activate
-    python3 setup.py develop
+    $ python3 -m virtualenv ./venv
+    $ source venv/bin/activate
+    $ (venv) python setup.py develop
```

Comment:
This is a little confusing to copy paste but I get the point of showing the command line more accurately. I think there is a balance of making steps easy to copy paste and showing python code as exactly as you would see

Reply:
This isn't Python code, these are shell commands. (The "$" in front is intended to represent the shell prompt.) I don't know how this can be clarified further.